test/functions/resolution/errors/testBadCallCandidates.chpl

```diff
@@ -1,5 +1,5 @@
 config param testNum = 1;
-
+/* Note: some cases commented-out due to stopgap scope resolver ambiguity */
```

Comment:
It would be good to note this change on some issue or somewhere to make sure we come back to this test to uncomment the cases.

Reply:
I've created #23085 to track this future work and to note that this test should change when it is complete.